base/checked.jl

```diff
@@ -131,20 +131,20 @@ checked_abs(x::Bool) = x
 Calculates `r = x+y`, with the flag `f` indicating whether overflow has occurred.
 """
 function add_with_overflow end
-add_with_overflow{T<:SignedInt}(x::T, y::T)   = checked_sadd_int(x, y)
-add_with_overflow{T<:UnsignedInt}(x::T, y::T) = checked_uadd_int(x, y)
-add_with_overflow(x::Bool, y::Bool)           = x+y, false
+add_with_overflow(x::T, y::T) where {T<:SignedInt}   = checked_sadd_int(x, y)
+add_with_overflow(x::T, y::T) where {T<:UnsignedInt} = checked_uadd_int(x, y)
+add_with_overflow(x::Bool, y::Bool) = (x+y, false)
```

Comment:
that's what I figured... I guess I will change them all back to no tusing extra spaccing for the Bool cases